### PR TITLE
Passed AdaptiveCardsHostConfig as a prop to CommonCard

### LINKED ISSUE #2106: AdaptiveCardHostConfig is not being passed to CommonCard component as a prop
## Screenshots

image

## Version
Local build: 0.0.0-0

## Describe the bug
Rich Cards that are built on top of the CommonCard Component - AnimationCards, AudioCards, SignInCards, and VideoCards - are not passing the AdaptiveCardHostConfig as prop to CommonCard. Web Chat is raising a "The prop \`adaptiveCardHostConfig\` is marked as required in \`CommonCard\`, but its value is \`undefined\`." warning as a result. 

## To Reproduce

Send one of the cards mentioned above as an attachment to Web Chat. 

```javascript
await context.sendActivity({ attachments: [
  CardFactory.animationCard(
     'Microsoft Bot Framework',
      [ { url: 'https://i.giphy.com/Ki55RUbOV5njy.gif' }],
      [],
      { subtitle: 'Animation Card' });
  ] 
});
```

[Bug]

### PR DESCRIPTION
**Pull request by tdurnford**
Fixes #2106

## Changelog Entry
Fix [#2106](https://github.com/Microsoft/BotFramework-WebChat/issues/2016). Fixed AdaptiveCardHostConfig warning associated with the CommonCard component, by [@tdurnford](https://github.com/tdurnford) in PR [#2108](https://github.com/Microsoft/BotFramework-WebChat/pull/2108)

## Specific Changes
I added the AdaptiveCardHostConfig as a prop to CommonCard in the AnimationCards, AudioCards, SignInCards, and VideoCards attachment components.  I also set the default value of the image in the AudioCardAttachment component to an empty object since that was throwing an error as well.

---

-  [ ] Testing Added

### CODE REVIEW

File: packages/bundle/src/adaptiveCards/Attachment/AnimationCardAttachment.js [hunk 1]
```diff
@@ -26,7 +26,13 @@ class AnimationCardAttachment extends React.Component {
   }
 
   render() {
-    const { adaptiveCards, attachment, attachment: { content: { media = [] } = {} } = {}, styleSet } = this.props;
+    const {
+      adaptiveCards,
+      adaptiveCardHostConfig,
```
Comment (corinagum) [inline packages/bundle/src/adaptiveCards/Attachment/AnimationCardAttachment.js]: sort alphabetical

Comment (tdurnford) [reply packages/bundle/src/adaptiveCards/Attachment/AnimationCardAttachment.js]: You know... I thought I was.... thanks for pointing that out! 🙃

File: packages/bundle/src/adaptiveCards/Attachment/AnimationCardAttachment.js [hunk 2]
```diff
@@ -41,7 +47,11 @@ class AnimationCardAttachment extends React.Component {
             </li>
           ))}
         </ul>
-        <CommonCard adaptiveCards={adaptiveCards} attachment={attachment} />
+        <CommonCard
+          adaptiveCards={adaptiveCards}
```
Comment (corinagum) [inline packages/bundle/src/adaptiveCards/Attachment/AnimationCardAttachment.js]: sort alphabetical

File: packages/bundle/src/adaptiveCards/Attachment/AudioCardAttachment.js [hunk 3]
```diff
@@ -10,8 +10,11 @@ const { AudioContent } = Components;
 
 const AudioCardAttachment = ({
   adaptiveCards,
```
Comment (corinagum) [inline packages/bundle/src/adaptiveCards/Attachment/AudioCardAttachment.js]: sort alphabetical

File: packages/bundle/src/adaptiveCards/Attachment/AudioCardAttachment.js [hunk 4]
```diff
@@ -22,7 +25,7 @@ const AudioCardAttachment = ({
         </li>
       ))}
     </ul>
-    <CommonCard adaptiveCards={adaptiveCards} attachment={attachment} />
+    <CommonCard adaptiveCards={adaptiveCards} adaptiveCardHostConfig={adaptiveCardHostConfig} attachment={attachment} />
```
Comment (corinagum) [inline packages/bundle/src/adaptiveCards/Attachment/AudioCardAttachment.js]: sort alphabetical

File: packages/bundle/src/adaptiveCards/Attachment/SignInCardAttachment.js [hunk 5]
```diff
@@ -4,9 +4,9 @@ import React from 'react';
 
 import CommonCard from './CommonCard';
 
-const SignInCardAttachment = ({ adaptiveCards, attachment, styleSet }) => (
+const SignInCardAttachment = ({ adaptiveCards, adaptiveCardHostConfig, attachment, styleSet }) => (
```
Comment (corinagum) [inline packages/bundle/src/adaptiveCards/Attachment/SignInCardAttachment.js]: lol same for the rest of these files xD

**Pull request merged**